nmdc_schema/migrators/migrator_from_X_to_PR2_and_PR24.py

```diff
@@ -0,0 +1,30 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+
+
+class Migrator(MigratorBase):
+    """
+    Migrates data between from X to PR2 and PR24
```

Comment:
I think there's a type here. Suggestion:

```diff
- Migrates data between from X to PR2 and PR24
+ Migrates data from X to PR2 and PR24
```

Reply:
Thanks Eric! I fixed it.